test/e2e/test/kibana/builder.go

```diff
@@ -106,6 +107,21 @@ func (b Builder) WithResources(resources corev1.ResourceRequirements) Builder {
 	return b
 }
 
+func (b Builder) WithMutatedFrom(mutatedFrom *Builder) Builder {
```

Comment:
The ES builder does not have this function, can you add it for consistency (or remove this one, up to you)?